pythonsd/sitemap.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime, timezone
+
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
+
+
+class IndexSitemap(Sitemap):
+	changefreq = 'weekly'
+	protocol = 'https'
+	lastmod = datetime(2024, 5, 15, 0, 0, 0, tzinfo=timezone.utc)
```

Comment:
I wonder if we should just remove this since it's going to quickly get out of date. It is [optional](https://docs.djangoproject.com/en/4.2/ref/contrib/sitemaps/#django.contrib.sitemaps.Sitemap.lastmod). Thoughts?

```suggestion
```